turborepo-tests/integration/tests/lockfile-aware-caching/berry.t

```diff
@@ -1,6 +1,7 @@
 Setup
   $ . ${TESTDIR}/../../../helpers/setup.sh
   $ . ${TESTDIR}/setup.sh $(pwd) berry
+  $ export COREPACK_ENABLE_DOWNLOAD_PROMPT=0
```

Comment:
I think it would make sense to put this in the `setup.sh` script instead of adding it adhoc to each test that might trigger `corepack`.

Reply:
sure! added in 6433e7b7a02f85f84084a93bc14cb1692dc02d5c